extensions/google-translate/src/translate.tsx

```diff
@@ -82,14 +82,13 @@ export default function Translate(): ReactElement {
             />
             {r.pronunciationText && (
               <List.Item
-                key={index}
                 title={r.pronunciationText}
                 accessories={[{ text: languages, tooltip: tooltip }]}
                 detail={<List.Item.Detail markdown={r.pronunciationText} />}
                 actions={
                   <ActionPanel>
                     <ActionPanel.Section>
-                      <Action.CopyToClipboard title="Copy" content={r.pronunciationText} />
+                      <ConfigurableCopyPasteActions value={r.translatedText} />
```

Comment:
the wrong value used, lets fix it:

```suggestion
                      <ConfigurableCopyPasteActions value={r.pronunciationText} />
```